src/shared/helpers.ts

```diff
@@ -121,3 +121,13 @@ export const getPermissionVariablesFromCookie = (req: RequestExtended): Permissi
   if (!permission_variables) throw new Error('No permission variables')
   return JSON.parse(permission_variables)
 }
+
+export const getGravatarUrl = (email: string) => {
```

Comment:
Please type this properly and allow a undefined value, this should make it easier to implement:

```ts
export const getGravatarUrl = (email: string | undefined): string | null => {
  if (!APPLICATION.GRAVATAR_ENABLE || !email) return null;

  return gravatar.url(email, {
    r: APPLICATION.RATING,
    protocol: 'https',
    default: APPLICATION.GRAVATAR_DEFAULT
  })
}
```